test/support/password_helpers.rb

```diff
@@ -0,0 +1,10 @@
+module PasswordHelpers
+  def password_visibility_toggle_test(container_id)
+    within(container_id) do
```

Comment:
can we search the div that contains a field with the label we expect, so we don't have to add an id to the container?